src/windows-current-connections.js

```diff
@@ -1,32 +1,64 @@
+const os = require('os');
 const execFile = require('child_process').execFile;
 const env = require('./env');
 const networkUtils = require('./utils/network-utils.js');
 
+function isWin11() {
+  const winMajor = os.release().split('.')[0];
+  const winMinor = os.release().split('.')[1];
+  const winBuild = os.release().split('.')[2];
+  return winMajor >= 10 && winMinor >= 0 && winBuild >= 22000;
```

Comment:
Shouldn't this be `winMajor >= 11`? It still works because there is also the winBuild restriction though.

Reply:
Answering my own question: no, winMajor is still 10 in windows 11:

https://stackoverflow.com/a/76873811